Makefile

```diff
@@ -14,49 +14,82 @@ build:
 ansible-image:
 	@bash ./scripts/build_image.sh
 
-# Brings up a demo cluster to install Contiv on - by default this is a docker, centos cluster.
-# It can be configured to start a RHEL cluster by setting CONTIV_NODE_OS=rhel7.
-# It can be started with k8s kubeadm install by running with CONTIV_KUBEADM=1.
-cluster: cluster-destroy
-	cd cluster && vagrant up
+# Brings up a demo cluster to install Contiv on with docker, centos.
+cluster-docker: 
```

Comment:
please rename these to match existing code/doc:

cluster-docker -> cluster-legacy-swarm, here we install docker, swarm, and contiv
cluster-swarm -> cluster-swarm-mode, here we only install contiv